frodo.js

```diff
@@ -10,8 +10,6 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.